classes/class-wc-connect-help-provider.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
+
+	class WC_Connect_Help_Provider {
+
+		/**
+		 * @var WC_Connect_Service_Settings_Store
+		 */
+		protected $service_settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $service_settings_store ) {
+
+			$this->service_settings_store = $service_settings_store;
+
+			add_filter( 'woocommerce_admin_status_tabs', array( $this, 'status_tabs' ) );
+			add_action( 'woocommerce_admin_status_content_connect', array( $this, 'page' ) );
+
+		}
+
+		/**
+		 * Filters the WooCommerce System Status Tabs to add connect
+		 *
+		 * @param array $tabs
+		 * @return array
+		 */
+		public function status_tabs( $tabs ) {
+
+			if ( ! is_array( $tabs ) ) {
+				$tabs = array();
+			}
+			$tabs[ 'connect' ] = _x( 'Connect', 'The WooCommerce Connect brandname', 'woocommerce' );
```

Comment:
I'm 50/50 on this one but I think this maybe should say "WooCommerce Connect" despite being somewhat redundant. Just to conserve the "brandname"